tests/FSharp.Compiler.ComponentTests/Conformance/DeclarationElements/MemberDefinitions/MethodsAndProperties/E_Abstract_ReusedParam.fs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+type I =
```

Comment:
Add a test for signature files please

Reply:
In my last commit I have added a test that checks a signature file. But I was not sure if the expectation was to get twice compiler errors as we know are checking both signatures and implementation files

Reply:
Yes, it's ok to get the errors twice